backend/src/org/commcare/util/CommCarePlatform.java

```diff
@@ -170,4 +172,13 @@ public String getMenuDisplayStyle(String menuId) {
         }
         return commonDisplayStyle;
     }
+
+    public OfflineUserRestore getDemoUserRestore() {
+        return offlineUserRestore;
+    }
+
+    @Override
+    public void registerDemoUserRestore(OfflineUserRestore offlineUserRestore) {
```

Comment:
Not sure I'm following why it's dangerous. Agreed that the spec doesn't say this explicitly right now, but it seems reasonable to add that as a constraint?